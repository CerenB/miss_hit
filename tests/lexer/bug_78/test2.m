% (c) Copyright 2020 Florian Schanda

disp x(2,N]
