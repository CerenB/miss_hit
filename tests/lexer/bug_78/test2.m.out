In test2.m, line 1
| % (c) Copyright 2020 Florian Schanda
| ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ info: [SL] COMMENT
In test2.m, line 1
| % (c) Copyright 2020 Florian Schanda
|                                     ^ info: NEWLINE
In test2.m, line 3
| disp x(2,N]
| ^^^^ info: [SL] IDENTIFIER
In test2.m, line 3
| disp x(2,N]
|      ^^^ info: CARRAY
In test2.m, line 3
| disp x(2,N]
|         ^ info: COMMA
In test2.m, line 3
| disp x(2,N]
|          ^ info: [S ] IDENTIFIER
In test2.m, line 3
| disp x(2,N]
|           ^ [31;1mlex error[0m: unmatched ]
In test2.m, line 3
| disp x(2,N]
|           ^ info: M_KET
In test2.m, line 3
| disp x(2,N]
|            ^ info: NEWLINE
test2.m: lexed OK
MISS_HIT Summary: 1 file(s) analysed, 1 error(s)
